dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -84,10 +84,11 @@
 
   private static final Logger logger = Logger.getLogger(RepositoryUtility.class.getName());
   
-  public static final RemoteRepository CENTRAL =
-      new RemoteRepository.Builder("central", "default", "http://repo1.maven.org/maven2/").build();
+  public static final RemoteRepository GOOGLE_MIRROR =
```

Comment:
Adding the Google mirror is fine, but I'm not sure we want to remove the central repository.

Reply:
This change is not to be merged. I'm experimenting how it solves (or not).